modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -132,19 +135,19 @@ spec:
                   number: 80
 ----
 <1> Defines the ingress resource.
-<2> Specifies the name of the ingress resource.
-<3> Specifies the name of the service resource.
+<2> Specifies the ingress name.
```

Comment:
Is Ingress supposed to be capital? I saw some instances where you made the lowercase capital and didn't know if that applies here.

Reply:
Nice catch! I updated "ingress" instances per https://redhat-documentation.github.io/supplementary-style-guide/#ingress